solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java

```diff
@@ -340,6 +347,12 @@ protected void setParser(ResponseParser parser) {
 
   protected abstract void updateDefaultMimeTypeForParser();
 
+  /** Experimental; subject to change! */
```

Comment:
[Q] Why is this deprecated the moment it's being added?

[0] If we want this to be experimental, can we use the `@lucene.experimental` annotation to track that as well?